src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -161,6 +161,37 @@ describe("calcite-input-time-picker", () => {
     expect(inputTimePickerValue).toBe(expectedValue);
   });
 
+  it("value displays correctly in the input when it is programmatically changed for a 12-hour language when a default value is present", async () => {
```

Comment:
Does this need a test for the 24-hour format case?

Reply:
Sure couldn't hurt, I'll add that!